Makefile

```diff
@@ -36,6 +36,24 @@ gotest:
 
 test: generate lint govet gotest
 
+# all .go files in the ecs-init
+GOFILES:=$(shell go list -f '{{$$p := .}}{{range $$f := .GoFiles}}{{$$p.Dir}}/{{$$f}} {{end}}' ./ecs-init/...)
+
+.PHONY: gocyclo
+gocyclo:
+	# Run gocyclo over all .go files
+	gocyclo -over 15 ${GOFILES}
```

Comment:
we could dial this down a bit for init package I think?

Reply:
currently the max value in the code base is 12, will make it 12.